examples/gallery/histograms/histogram.py

```diff
@@ -4,7 +4,8 @@
 The :meth:`pygmt.Figure.histogram` method can plot regular histograms.
 Using the ``series`` parameter allows to set the interval for the width of
 each bar. The type of the histogram (frequency count or percentage) can be
-selected via the ``histtype`` parameter.
+selected via the ``histtype`` parameter. The ``fill`` of the
+bars can be either a standard color name or a pattern.
```

Comment:
Maybe we should simply write "color" as beside one of the GMT standard color names users can pass a string for R/G/B levels (e.g., `"100/123/52"`) or hex value (e.g., `"#333333"`) even we are not showing this explicitly in this gallery example?

```suggestion
selected via the ``histtype`` parameter. The ``fill`` of the
bars can be either a color or a pattern.
```